packages/client/composables/useNav.ts

```diff
@@ -32,6 +32,9 @@ export function useNav(route: ComputedRef<RouteRecordRaw | RouteLocationNormaliz
     route,
     path,
     total,
+    clicksContext,
+    clicks,
+    clicksTotal,
```

Comment:
Is it ok to use global state rather than the one provided in injection? Wouldn't that make dynamic condition in overviews broken?